src/app/weather-data/weather-data.component.spec.ts

```diff
@@ -38,4 +38,73 @@ describe('WeatherDataComponent', () => {
     fixture.detectChanges();
     expect(fixture).toMatchSnapshot();
   });
+
+  describe('Should test windChecker function', () => {
+    it('Should return True', () => {
+      component.checkAttributes = [];
+      let data = 2;
```

Comment:
Are you expecting this variable to change its contents? Consider using `const` instead of `let`? Might need to apply this to other `data` variables.